databricks/koalas/series.py

```diff
@@ -60,9 +60,10 @@
 from databricks.koalas.ml import corr
 from databricks.koalas.utils import (
     validate_arguments_and_invoke_function,
-    scol_for,
     combine_frames,
     name_like_string,
+    same_anchor,
+    scol_for,
```

Comment:
It's really not a big deal, but I'm just curious that why the position of `scol_for` has been moved?

Reply:
The order should basically be alphabetical order.
I'd like black or something to manage the order..

Reply:
Ah, I got it! Sorry for annoying and thanks for the reply !